sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_recognize_identity_documents_async.py

```diff
@@ -72,9 +72,9 @@ async def recognize_identity_documents(self):
                 address = id_document.fields.get("Address")
                 if address:
                     print("Address: {} has confidence: {}".format(address.value, address.confidence))
-                country = id_document.fields.get("Country")
+                country = id_document.fields.get("CountryRegion")
                 if country:
-                    print("Country: {} has confidence: {}".format(country.value, country.confidence))
+                    print("Country Region: {} has confidence: {}".format(country.value, country.confidence))
```

Comment:
Do we need to add a slash here? Or join the words like the field name?

Reply:
I was wondering if `Country Region` should be `Country/Region` or `CountryRegion`? Simply because I skimmed through the docs shared from the service team and I know it is sensitive to how this information is shown. 

Ok just checked the doc...I think we should go with `Country/Region` to make sure we're displaying the information correctly (this is the example I see the guidance following the most)

Reply:
Honestly this feels like a nit to me 😅 but just to be said since I'm not sure if the space implies something different than the slash...but I may be analyzing this too much.